pkg/sidecar/k8s_reactor.go

```diff
@@ -37,17 +37,26 @@ var (
 )
 
 type K8sReactor struct {
-	client  *sync.Client
-	redis   net.IP
-	manager *docker.Manager
+	client         *sync.Client
+	manager        *docker.Manager
+	servicesRoutes []net.IP
 }
 
 func NewK8sReactor() (Reactor, error) {
-	redisHost := os.Getenv(EnvRedisHost)
+	// TODO: Generalize this to a list of services.
+	wantedRoutes := []string{
+		os.Getenv(EnvRedisHost),
+		os.Getenv(EnvInfluxdbHost),
+	}
 
-	redisIp, err := net.ResolveIPAddr("ip4", redisHost)
-	if err != nil {
-		return nil, fmt.Errorf("failed to resolve redis: %w", err)
+	var resolvedRoutes []net.IP
+	for _, route := range wantedRoutes {
+		ip, err := net.ResolveIPAddr("ip4", route)
+		if err != nil {
```

Comment:
Change in behaviour here - we no longer return an error if we can't resolve the service, allowing for `optional` services.